crates/bioauth-flow-rpc/src/errors/robonode_request.rs

```diff
@@ -0,0 +1,16 @@
+//! The robonode requests related error.
+
+use rpc_validator_key_logic::Error as ValidatorKeyError;
+
+use super::sign::Error as SignError;
+
+/// The robonode requests related error.
+#[derive(Debug)]
+pub enum Error<T: std::error::Error + 'static> {
```

Comment:
```suggestion
pub enum FlowBaseError<T: std::error::Error + 'static> {
```
If we take https://github.com/humanode-network/humanode/pull/1026/files#r1719196297 into consideration

Reply:
Improved